tests/basilisp/test_io.lpy

```diff
@@ -203,7 +208,7 @@
       (try
         (spit-bytes filename "hi there")
         (is (= (bytes "hi there") (slurp-bytes filename)))
-        (is (= (bytes "hi there") (slurp-bytes (str "file://" filename))))
+        (is (= (bytes "hi there") (slurp-bytes (str "file:///" filename))))
```

Comment:
This won't affect MacOS or Linux?

Reply:
It actually makes it compliant with the URI file spec which is `file://host/path`, where the `host` is empty:

https://en.wikipedia.org/wiki/File_URI_scheme

The correctness of this will be evident in CI where the tests should be passing on ubuntu